CHANGELOG.md

```diff
@@ -1,5 +1,58 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.10.3
+
+_Aug 29, 2022_
+
+A big thanks to the 15 contributors who made this release possible. This release was mostly around 🐛 bug fixes and 📚 documentation improvements.
```

Comment:
I guess we can highlight https://github.com/mui/material-ui/pull/32735 now that it is merged. The versions looks good, but we will need to bump the version of @mui/types too.